persistent-template/Database/Persist/TH.hs

```diff
@@ -167,9 +153,6 @@ persistFileWith ps fp = persistManyFileWith ps [fp]
 -- @since 2.5.4
 persistManyFileWith :: PersistSettings -> [FilePath] -> Q Exp
 persistManyFileWith ps fps = do
-#ifdef GHC_7_4
-    mapM_ qAddDependentFile fps
-#endif
```

Comment:
I think we want to keep this line. If I read the original code right, this line of code only gets included if the GHC version is `>= 7.4`. We're definitely doing that, so we want to keep this line.

Reply:
Ah, good catch, probably misread it as `if ghc == 7.4`

Reply:
CPP should be bant

Reply:
bant?

Reply:
It's a dumb way of saying "banned." I really dislike how difficult it makes ot understand and use the codebase.